apps/events-helsinki/src/pages/articles/index.tsx

```diff
@@ -65,6 +65,10 @@ export default function ArticleArchive({
     networkStatus,
   } = usePostsQuery({
     notifyOnNetworkStatusChange: true,
+    // does not work without fetchPolicy: 'network-only',
+    fetchPolicy: 'network-only',
+    // not sure if nextFetchPolicy is needed
+    nextFetchPolicy: 'cache-first',
```

Comment:
This looks OK to me.

fetchPolicy is used every time the component mounts. NextFetchPolicy is used for every query execution after the component mounts. 

https://www.apollographql.com/docs/react/data/queries/#nextfetchpolicy
> You can also specify a query's nextFetchPolicy. If you do, fetchPolicy is used for the query's first execution, and nextFetchPolicy is used to determine how the query responds to future cache updates